src/module-elasticsuite-catalog-optimizer/etc/logging.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<logging xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Logging:etc/logging.xsd">
+    <groups>
+        <group name="elasticsuite_optimizer">
+            <label translate="true">Manage Elasticsuite Search Optimizers</label>
```

Comment:
Please add i18n files for en_US and fr_FR.
Translation for fr_FR is "Manage Elasticsuite Search Optimizers","Gérer les optimiseurs Elasticsuite"